ll-list-control.html

```diff
@@ -117,34 +129,30 @@
       // Use this to clean up anything you did in `attached`.
     },
 
-    // Element Behavior
-
-    /**
-     * The `ll-list-control-lasers` event is fired whenever `fireLasers` is called.
-     *
-     * @event ll-list-control-lasers
-     * @detail {{sound: String}}
-     */
-
-    /**
-     * Sometimes it's just nice to say hi.
-     *
-     * @param {string} greeting A positive greeting.
-     * @return {string} The full greeting.
-     */
-    sayHello: function(greeting) {
-      var response = greeting || 'Hello World!';
-      return 'll-list-control says, ' + response;
+    _showEmptyText: function(){
+      if(this.items && this.items.length > 0){
+        this.$.listEmpty.style.display = "none";
```

Comment:
Might just be a preference, but it seems cleaner to me to add a class and then have CSS style the display. You can use:

this.toggleClass

https://www.polymer-project.org/1.0/docs/devguide/utility-functions.html

so in this case, we could do this.toggleClass('hidden', true,  this.$.listEmpty) to add the hidden class.